drink-cli/src/executor/mod.rs

```diff
@@ -68,11 +68,13 @@ pub fn execute(app_state: &mut AppState) -> Result<()> {
 
 fn build_blocks(app_state: &mut AppState, count: u64) {
     for _ in 0..count {
-        app_state.session.chain_api().build_block();
+        app_state.chain_info.block_height = app_state
```

Comment:
Any reason why we're not using the `ChainApi::build_block` method?

Reply:
nice spot, changed to `build_blocks`